<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PunchPropertiesClass</class>
 <widget class="QWidget" name="PunchPropertiesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PunchProperties</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Printer</string>
   </property>
   <widget class="QComboBox" name="punchCombo">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>3525</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Device Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deviceNumber">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>226</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Device Type</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>431</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Optional Arguments</string>
   </property>
   <widget class="QLineEdit" name="fileName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OKButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>78</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>78</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="noclearCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>84</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Append</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="crlfCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>84</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Crlf</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="asciiCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>84</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Ascii</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
